addons/openebs/template/testgrid/k8s-docker.yaml

```diff
@@ -0,0 +1,9 @@
+kubernetes:
+  version: "latest"
+weave:
+  version: "latest"
+docker:
+  version: "latest"
+openebs:
+  version: "__testver__"
+  s3Override: "__testdist__"
```

Comment:
IMO we should also include minio or registry here to use a PVC and ensure openebs is working

Reply:
adding minio